src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,32 @@
+import game.characters.Knight
+import game.characters.Warrior
+import game.collections.Army
+import game.interactions.Battle
+import game.interfaces.BaseWarrior
+import game.settings.Params
+
+
+fun main() {
+
+
+ fun smokeTestOne() {
```

Comment:
why do we need this `fun` inside another `fun` here ?

Reply:
Removed the unnecessary inner fun.